app/graphql/schema.rb

```diff
@@ -18,4 +18,18 @@ class Schema < GraphQL::Schema
   use GraphQL::Execution::Interpreter
   use GraphQL::Analysis::AST
   disable_introspection_entry_points if Rails.env.production?
+
+  # rubocop:disable Layout/ArgumentAlignment
+  # rubocop:disable Layout/LineLength
+  def self.unauthorized_object(error)
+    raise GraphQL::ExecutionError,
+      "An object of type #{error.type.graphql_name} was hidden due to permissions"
+  end
```

Comment:
I'd prefer a custom exception related to the authorization. Also you don't really need the message as you're catching it later and drop one there.